platform/windows/Corona.Native.Library.Win32/Interop/Input/Key.cpp

```diff
@@ -362,6 +363,38 @@ const char* Key::GetCoronaName() const
 	return fCoronaName;
 }
 
+const char* Key::GetCharacter()
+{
+	unsigned char *keyboardState = new unsigned char[256];
+	GetKeyboardState(keyboardState);
+
+	int scanCode = MapVirtualKey(this->GetNativeCodeValue(), 0x0); // 0x0 is MAPVK_VK_TO_VSC
+	if (scanCode == 0)
+	{
+		return NULL;
+	}
+	else
+	{
+		const int BUFFER_LENGTH = 2;
+		wchar_t chars[BUFFER_LENGTH];
+		chars[BUFFER_LENGTH - 1] = 0;
+
+		switch (ToUnicode(this->GetNativeCodeValue(), scanCode, keyboardState, chars, BUFFER_LENGTH - 1, 0))
+		{
+		case -1:
+		case 0:
+			return NULL;
+		default:
+			WinString stringConverter;
+			stringConverter.SetUTF16(chars);
+			int utf8Length = strlen(stringConverter.GetUTF8()) + 1;
+			char * out = new char[utf8Length];
```

Comment:
This is a memory leak it seems. Probably should wrap it in std::string or something